### ISSUE TITLE
Fragment caching not working when django debug toolbar is enabled.

### ISSUE THREAD

**Issue by pstreck** [jazzband/django-debug-toolbar#877]
When working in Django 1.10 or 1.10.1 fragment caching is not working with Django Debug Toolbar enabled. View caching and ImageKit caching work fine though.

**Comment by matthiask** [comment]
I'm not sure what you mean by fragment caching. Are you saying that we break the `{% cache %}` template tag in some way?

**Comment by pstreck** [comment]
Yes, exactly. With the toolbar enabled elements within the {% cache %} {% endcache %} tags are not cached in my backend (memcached). With the toolbar disabled everything is cached as expected.

**Comment by pstreck** [comment]
Is there any information I can provide to help debug this incident? I would like to help resolve it if possible.

**Comment by millin** [comment]
For me caching works fine (I checked it from memcached side), but cache panel of django-debug-toolbar doesn't show any calls.

**Comment by zetaab** [comment]
I have same issue, cache works correctly but django-debug-toolbar does not show any calls.

**Comment by r3m0t** [comment]
This is the same as #799 ,  `caches` needs to be monkey-patched in `django.templatetags.cache`
